otlp/traces.go

```diff
@@ -125,6 +120,21 @@ func TranslateTraceRequest(request *collectorTrace.ExportTraceServiceRequest, ri
 					}
 					if isError {
 						attrs["error"] = true
+
+						// Copy over span event error attributes because:
+						// 1. They are common and high-value for error investigations
+						// 2. It sucks to have to look at span events in our trace UI today to hunt these down on an error span
+						// 3. This makes bubble up better because you can see these error details without having to query the span events
+						for _, seventAttr := range sevent.Attributes {
+							if seventAttr.Key == "exception.message" ||
+								seventAttr.Key == "exception.type" ||
+								seventAttr.Key == "exception.stacktrace" ||
+								seventAttr.Key == "exception.excaped" {
+								if _, present := attrs[seventAttr.Key]; !present {
+									attrs[seventAttr.Key] = seventAttr.Value
```

Comment:
`attrs` is the span event's attributes.  To set the attributes for the Honeycomb event representing the span we need to use `eventAttrs` right?

Reply:
whoopsie